### DISCUSSION TITLE
flight controller velox cine f7

### DISCUSSION THREAD

**Discussion by robert1739q** [iNavFlight/inav#9892] [Support]
how do i go about asking for a target for the t-motor velox cine f7 / so i can use it in inav 7.0

**Comment by sensei-hacker** [comment]
The first, best place to start is to contact the manufacturer.

**Comment by sensei-hacker** [comment]
Unless that board is the TMOTORVELOXF7V2 target?

**Reply by nick-scott** [reply]
Maybe? Supposedly Tmotor reached out last year to get this in the works
https://github.com/iNavFlight/inav/issues/9198

Also this guy said he had a F7 CINE board, so I think you're right
https://github.com/iNavFlight/inav/issues/9186
